package.json

```diff
@@ -40,30 +35,32 @@
   "dependencies": {
     "aegir": "^20.3.1",
     "base64url": "^3.0.0",
-    "bl": "^3.0.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "cids": "~0.7.1",
-    "cross-env": "^5.2.0",
+    "cross-env": "^6.0.3",
     "delay": "^4.3.0",
     "detect-node": "^2.0.4",
     "dirty-chai": "^2.0.1",
     "hat": "~0.0.3",
-    "ipfs-unixfs": "~0.1.16",
-    "ipfsd-ctl": "~0.47.4",
-    "ipns": "^0.6.1",
+    "ipfs-unixfs": "^0.3.0",
+    "ipfsd-ctl": "github:ipfs/js-ipfsd-ctl#remove-option-normalisation",
+    "ipns": "^0.7.0",
     "is-ci": "^2.0.0",
     "is-os": "^1.0.1",
-    "libp2p-websocket-star-rendezvous": "~0.4.1",
-    "merge-options": "^1.0.1",
+    "it-all": "^1.0.1",
+    "it-concat": "^1.0.0",
+    "it-last": "^1.0.1",
+    "libp2p-webrtc-star": "^0.17.3",
     "mocha": "^5.2.0",
     "multihashes": "~0.4.14",
     "p-retry": "^4.1.0",
     "pretty-bytes": "^5.1.0",
     "promisify-es6": "^1.0.3",
     "random-fs": "^1.0.3",
-    "readable-stream-buffer-stream": "~0.0.2",
-    "rimraf": "^2.6.3"
+    "readable-stream-buffer-stream": "github:alanshaw/readable-stream-buffer-stream#feat/browser-friendly",
```

Comment:
```suggestion
    "readable-stream-buffer-stream": "^1.0.0",
```

Reply:
Pushed this update (needs package-lock.json update as well) - thanks.